tests/dat/manifest_validate_action_annotations.yaml

```diff
@@ -0,0 +1,65 @@
+packages:
+    packageActionAnnotations:
+        actions:
+            helloworld:
+                function: actions/hello.js
+                runtime: nodejs:6
+                inputs:
+                    name:
+                        type: string
+                        description: name of a person
+                    place:
+                        type: string
+                        description: location of a person
+                outputs:
+                    payload:
+                        type: string
+                        description: a simple greeting message, Hello World!
+                annotations:
+                    action_annotation_1: this is annotation 1
```

Comment:
Do we have a test for more complex (i.e., arrays/JSON) annotations?  Should we open another issue (feature) to support this?

Reply:
Nope, created new issue https://github.com/apache/incubator-openwhisk-wskdeploy/issues/634